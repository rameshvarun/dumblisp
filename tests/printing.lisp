; A basic Hello World example
(print "Hello World!")
(print "This is dumblisp:" 100)
(print "This is a booelan:" false TRUE)
