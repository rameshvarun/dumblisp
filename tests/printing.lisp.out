Hello World!
This is dumblisp: 100
This is a booelan: FALSE TRUE
